cypress/e2e/article.cy.js

```diff
@@ -1,24 +1,74 @@
 /// <reference types="cypress" />
 /// <reference types="../support" />
 
-describe('Article', () => {
-  before(() => {
+import NewArticlePageObject from '../support/pages/newArticle.pageObject';
+import ArticlePageObject from '../support/pages/article.pageObject';
+import HomePageObject from '../support/pages/home.pageObject';
 
-  });
+const publishArticle = new NewArticlePageObject();
+const articlePage = new ArticlePageObject();
+const homePage = new HomePageObject();
+
+const testData = {
+  articleTitle: 'edited_title',
+  articleDescription: 'edited_description',
+  articleBody: 'edited_body'
+};
+
+describe('Article', () => {
+  let article;
 
   beforeEach(() => {
     cy.task('db:clear');
+    cy.wait(500);
+
+    cy.login();
+    cy.task('generateArticle').then(generateArticle => {
+      article = generateArticle;
+    });
+    publishArticle.visit();
   });
 
   it('should be created using New Article form', () => {
```

Comment:
You should consider creating a separate function to avoid code duplication in your tests. The code for creating new article is repeated in each test.